blocked-edges/4.12.0-rc.0-AWSOldBootImage.yaml

```diff
@@ -0,0 +1,21 @@
+to: 4.12.0-rc.0
+from: 4[.]11[.].*
+url: https://issues.redhat.com/browse/COS-1942
+name: AWSOldBootImages
+message: |-
+  4.2 AWS boot images are not compatible with 4.12.0-rc.0, and machines created with them will fail to become nodes.
+matchingRules:
+- type: PromQL
+  promql:
+    promql:
+      topk(1,
+        label_replace(group(cluster_version{type="initial",version=~"4[.][0-9][.].*"}),"born_by_4_9", "yes, so possibly actually born in 4.1 or 4.2", "", "")
+        or
+        label_replace(0 * group(cluster_version{type="initial",version!~"4[.][0-9][.].*"}),"born_by_4_9", "no, born in 4.10 or later", "", "")
+      )
+      * on () group_left (type)
+      (
+        cluster_infrastructure_provider{type="AWS"}
+        or
+        0 * cluster_infrastructure_provider
+      )
```

Comment:
Checked this on b01 (affected), b02 (not affected, gcp) and b05(not affected, installed as 4.10.9) and the query gives results as expected.